documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -94,10 +105,52 @@ m¦excludedBrokersForReplicaMove
 
 |===
 
+
+[discrete]
+=== Broker load
+
+The broker load is stored in a ConfigMap (with the same name as the KafakRebalance custom resource) as a JSON formatted string. This JSON string consists of a JSON object with keys for each broker IDs linking to a number of metrics for each broker.
+Each metric consist of three values.
+The first is the metric value before the optimization proposal is applied, the second is the expected value of the metric after the proposal is applied, and the third is the difference between the first two values (after minus before).
+
+To extract the JSON string from the ConfigMap, use the following command:
+
+[source,shell,subs=+quotes]
+----
+kubectl get configmap _my-rebalance_ -o json | jq '.["data"]["brokerLoad.json"]|fromjson|.'
```

Comment:
I am still not clear on why we are adding emphasis to text in a verbatim code example? When rendered, if I copy code would I get the markup as well?

Reply:
I tried this but  rendering we are not getting the the markup, but before removing them I guess we should confirm it with @laidan6000 .

Reply:
The markup would not be copied. @ShubhamRwt - Please use capitals and hyphens, so that would be:

MY-REBALANCE

Apologies for the confusion.